reference/api/settings.mdx

```diff
@@ -684,13 +684,43 @@ You can use the returned `taskUid` to get more details on [the status of the tas
 
 ## Filterable attributes
 
-Attributes in the `filterableAttributes` list can be used as filters or facets.
+Attributes in the `filterableAttributes` list can be used as [filters](/learn/filtering_and_sorting/filter_search_results) or [facets](/learn/filtering_and_sorting/search_with_facet_filters).
 
 <Capsule intent="warning">
-Updating filterable attributes will re-index all documents in the index, which can take some time. We recommend updating your index settings first and then adding documents as this reduces RAM consumption.
+Updating filterable attributes will re-index all documents in the index, which can take some time. To reduce RAM consumption, first update your index settings and then add documents.
 </Capsule>
 
-[To learn more about filterable attributes, refer to our dedicated guide.](/learn/filtering_and_sorting/filter_search_results)
+### Filterable attribute object
+
+`filterableAttributes` may be either an array of strings or an array of filterable attribute objects.
+
+Filterable attribute objects must contain the following fields:
+
+| Name                    | Type             | Default value | Description                                              |
+| ----------------------- | ---------------- | ------------- | -------------------------------------------------------- |
+| **`attributePatterns`** | Array of strings | `[]`          | A list of strings indicating filterable fields                      |
+| **`features`**          | Object           | `{}`          | A list outlining filter types enabled for the specified attributes                      |
+
+#### `attributePatterns`
+
+Attribute patterns may begin or end with a * wildcard to match multiple fields: `customer_*`, `attribute*`.
+
+You may also set attributePatterns to *. In this case Meilisearch will treat all fields as filterable.
```

Comment:
We may want to discourage this, this would imply unnecessary indexing time and disk consumption.